docs/how-tos/using-the-event-bus.rst

```diff
@@ -0,0 +1,84 @@
+Using the Open edX Event Bus
```

Comment:
@bmtcril: I also created a document on [how to ensure event bus compatibility for new events](https://github.com/openedx/openedx-events/pull/428), what do you think about merging these two docs into one?

Reply:
I think they are probably good as they are, don't want to make them too long!

Reply:
Good call!